### PR TITLE
feat: add color-select component

### PR DESCRIPTION
**Pull request by olehrakwix**
move color options to a separate component

### CODE REVIEW

File: src/components/color-select/color-select.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import styles from './color-select.module.scss';
+
+export interface ColorSelectOption {
+    colorName: string;
+    color: string;
```
Comment (yurii-ve) [inline src/components/color-select/color-select.tsx]: `{name: string, value: string}` or `{label: string, value: string}`

Comment (olehrakwix) [reply src/components/color-select/color-select.tsx]: that is not a value. that is css representation of color.

colorName (comes from ProductChoice.description) is used as a value and display value in SDK. color (comes from ProductChoice.value) is used to render that color

Comment (yurii-ve) [reply src/components/color-select/color-select.tsx]: what does it mean 'css representation of color'?
I can see `backgroundColor: o.color` below.. what is `o.color` if not a color value?

Comment (yurii-ve) [reply src/components/color-select/color-select.tsx]: if property name `colorName` comes from SDK doesn't mean we can't rename it in scope of the `ColorSelect` component. There is no need adding prefix "color" to each property in the `ColorSelectOption`

Comment (olehrakwix) [reply src/components/color-select/color-select.tsx]: renamed to `name` and `hexValue` as discussed

**Review by yurii-ve — APPROVED** [review]

**Pull request merged**